canonical-web-and-design/maas-ui
# Connect release notifications setting to the API.

## Done
- Add a selector for the release notifications setting.
- Update the notifications form to get the value via the selector.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps

- You'll need to update your MAAS to master (or use Bolla, it should have the updated API by now).
- Visit /settings.
- Change the setting for release notifications.
- Refresh, the form should show the value correctly.

## Fixes

Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/2056.
